app/src/main/res/values/strings.xml

```diff
@@ -4,4 +4,18 @@
     <string name="file_chooser"></string>
     <string name="file_chooser_default"></string>
     <string name="file_chooser_custom"></string>
+    <string-array name="notification_settings">
+        <!--item>Every minute</item>
+        <item>Once in 5 minutes</item>
+        <item>Once in 10 minutes</item-->

```

Comment:
why is this removed?

Reply:
WorkManager can run only once in 15 minutes, any faster would require a foreground service. There could be some issues when the user plugs in the device